source/includes/steps-configure-user-metadata-import-export.yaml

```
@@ -40,9 +43,11 @@ content: |
        ]
      }
 
-  For custom JWT authentication, the metadata_fields object has an additional
-  property, ``name``, which represents the path to the field. The 
-  ``field_name`` represents the name of the field.
+  **Custom JWT authentication** 
+  
+  The metadata_fields object has an additional property. This property, 
```

again, metadata_fields isn't object, but array. also i think putting 'For custom JWT authentication' per how it was before adds clarity.

```suggestion
For custom JWT authentication, ``metadata_fields`` has an additional property. This property, 
```